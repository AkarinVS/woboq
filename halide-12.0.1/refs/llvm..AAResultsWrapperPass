<inh f='halide/llvm-install/include/llvm/Pass.h' l='298' c='llvm::FunctionPass'/>
<def f='halide/llvm-install/include/llvm/Analysis/AliasAnalysis.h' l='1191' ll='1205'/>
<size>40</size>
<doc f='halide/llvm-install/include/llvm/Analysis/AliasAnalysis.h' l='1189'>/// A wrapper pass to provide the legacy pass manager access to a suitably
/// prepared AAResults object.</doc>
<mbr r='llvm::AAResultsWrapperPass::AAR' o='256' t='std::unique_ptr&lt;AAResults&gt;'/>
<smbr r='llvm::AAResultsWrapperPass::ID' t='char'/>
<fun r='_ZN4llvm20AAResultsWrapperPassC1Ev'/>
<fun r='_ZN4llvm20AAResultsWrapperPass12getAAResultsEv'/>
<fun r='_ZNK4llvm20AAResultsWrapperPass12getAAResultsEv'/>
<fun r='_ZN4llvm20AAResultsWrapperPass13runOnFunctionERNS_8FunctionE'/>
<fun r='_ZNK4llvm20AAResultsWrapperPass16getAnalysisUsageERNS_13AnalysisUsageE'/>
