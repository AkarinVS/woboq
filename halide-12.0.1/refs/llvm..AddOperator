<inh f='halide/llvm-install/include/llvm/IR/Operator.h' l='412' c='llvm::ConcreteOperator'/>
<def f='halide/llvm-install/include/llvm/IR/Operator.h' l='426' ll='428'/>
<size>24</size>
