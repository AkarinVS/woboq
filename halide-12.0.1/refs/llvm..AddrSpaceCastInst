<inh f='halide/llvm-install/include/llvm/IR/InstrTypes.h' l='432' c='llvm::CastInst'/>
<def f='halide/llvm-install/include/llvm/IR/Instructions.h' l='5196' ll='5253'/>
<use f='halide/llvm-install/include/llvm/IR/Instructions.h' l='5202' c='_ZNK4llvm17AddrSpaceCastInst9cloneImplEv'/>
<use f='halide/llvm-install/include/llvm/IR/Operator.h' l='623'/>
<size>64</size>
<doc f='halide/llvm-install/include/llvm/IR/Instructions.h' l='5190'>//===----------------------------------------------------------------------===//
//                          AddrSpaceCastInst Class
//===----------------------------------------------------------------------===//

/// This class represents a conversion between pointers from one address space
/// to another.</doc>
<fun r='_ZNK4llvm17AddrSpaceCastInst9cloneImplEv'/>
<fun r='_ZN4llvm17AddrSpaceCastInstC1EPNS_5ValueEPNS_4TypeERKNS_5TwineEPNS_11InstructionE'/>
<fun r='_ZN4llvm17AddrSpaceCastInstC1EPNS_5ValueEPNS_4TypeERKNS_5TwineEPNS_10BasicBlockE'/>
<fun r='_ZN4llvm17AddrSpaceCastInst7classofEPKNS_11InstructionE'/>
<fun r='_ZN4llvm17AddrSpaceCastInst7classofEPKNS_5ValueE'/>
<fun r='_ZN4llvm17AddrSpaceCastInst17getPointerOperandEv'/>
<fun r='_ZNK4llvm17AddrSpaceCastInst17getPointerOperandEv'/>
<fun r='_ZN4llvm17AddrSpaceCastInst22getPointerOperandIndexEv'/>
<fun r='_ZNK4llvm17AddrSpaceCastInst18getSrcAddressSpaceEv'/>
<fun r='_ZNK4llvm17AddrSpaceCastInst19getDestAddressSpaceEv'/>
