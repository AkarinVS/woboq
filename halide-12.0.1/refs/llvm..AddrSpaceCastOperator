<inh f='halide/llvm-install/include/llvm/IR/Operator.h' l='412' c='llvm::ConcreteOperator'/>
<def f='halide/llvm-install/include/llvm/IR/Operator.h' l='621' ll='638'/>
<size>24</size>
<fun r='_ZN4llvm21AddrSpaceCastOperator17getPointerOperandEv'/>
<fun r='_ZNK4llvm21AddrSpaceCastOperator17getPointerOperandEv'/>
<fun r='_ZNK4llvm21AddrSpaceCastOperator18getSrcAddressSpaceEv'/>
<fun r='_ZNK4llvm21AddrSpaceCastOperator19getDestAddressSpaceEv'/>
