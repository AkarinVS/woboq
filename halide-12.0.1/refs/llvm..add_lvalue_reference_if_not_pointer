<def f='halide/llvm-install/include/llvm/Support/type_traits.h' l='44'/>
<def f='halide/llvm-install/include/llvm/Support/type_traits.h' l='46' ll='50'/>
<use f='halide/llvm-install/include/llvm/Support/Casting.h' l='45'/>
