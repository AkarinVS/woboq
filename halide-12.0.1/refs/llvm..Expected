<use f='halide/llvm-install/include/llvm/Support/raw_ostream.h' l='546' c='_ZN4llvm14raw_fd_ostream4lockEv'/>
<use f='halide/llvm-install/include/llvm/Support/raw_ostream.h' l='556' c='_ZN4llvm14raw_fd_ostream10tryLockForENSt6chrono8durationIlSt5ratioILl1ELl1000EEEE'/>
<def f='halide/llvm-install/include/llvm/Support/Error.h' l='439' ll='688'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='499' c='_ZN4llvm8ExpectedC1EONS_8ExpectedITL0__EEPNSt9enable_ifIXsr3std14is_convertibleIS2_T_EE5valueEvE4typeE'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='508' c='_ZN4llvm8ExpectedC1EONS_8ExpectedITL0__EEPNSt9enable_ifIXntsr3std14is_convertibleIS2_T_EE5valueEvE4typeE'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='545' c='_ZNK4llvm8Expected3getEv'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='599' c='_ZN4llvm8Expected13moveConstructEONS_8ExpectedITL0__EE'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='612' c='_ZN4llvm8Expected10moveAssignEONS_8ExpectedITL0__EE'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='736' c='_ZN4llvm8cantFailENS_8ExpectedIT_EEPKc'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='767' c='_ZN4llvm8cantFailENS_8ExpectedIRT_EEPKc'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='964' c='_ZN4llvm14handleExpectedENS_8ExpectedIT_EEOT0_DpOT1_'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='964' c='_ZN4llvm14handleExpectedENS_8ExpectedIT_EEOT0_DpOT1_'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='1017' c='_ZN4llvm18expectedToOptionalEONS_8ExpectedIT_EE'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='1084' c='_ZN4llvm22ExpectedAsOutParameterC1EPNS_8ExpectedIT_EE'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='1096'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='1142' c='_ZN4llvm17errorOrToExpectedEONS_7ErrorOrIT_EE'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='1149' c='_ZN4llvm17expectedToErrorOrEONS_8ExpectedIT_EE'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='1326' c='_ZNK4llvm11ExitOnErrorclEONS_8ExpectedIT_EE'/>
<use f='halide/llvm-install/include/llvm/Support/Error.h' l='1333' c='_ZNK4llvm11ExitOnErrorclEONS_8ExpectedIRT_EE'/>
<use f='halide/llvm-install/include/llvm/Object/Binary.h' l='182' c='_ZN4llvm6object12createBinaryENS_15MemoryBufferRefEPNS_11LLVMContextEb'/>
<use f='halide/llvm-install/include/llvm/Object/Binary.h' l='234' c='_ZN4llvm6object12createBinaryENS_9StringRefEPNS_11LLVMContextEb'/>
<use f='halide/llvm-install/include/llvm/Object/SymbolicFile.h' l='131' c='_ZNK4llvm6object14BasicSymbolRef8getFlagsEv'/>
<use f='halide/llvm-install/include/llvm/Object/SymbolicFile.h' l='149' c='_ZNK4llvm6object12SymbolicFile14getSymbolFlagsENS0_11DataRefImplE'/>
<use f='halide/llvm-install/include/llvm/Object/SymbolicFile.h' l='162' c='_ZN4llvm6object12SymbolicFile18createSymbolicFileENS_15MemoryBufferRefENS_10file_magicEPNS_11LLVMContextEb'/>
<use f='halide/llvm-install/include/llvm/Object/SymbolicFile.h' l='166' c='_ZN4llvm6object12SymbolicFile18createSymbolicFileENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Object/SymbolicFile.h' l='198' c='_ZNK4llvm6object14BasicSymbolRef8getFlagsEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='96' c='_ZNK4llvm6object10SectionRef7getNameEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='100' c='_ZNK4llvm6object10SectionRef11getContentsEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='135' c='_ZNK4llvm6object10SectionRef19getRelocatedSectionEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='183' c='_ZNK4llvm6object9SymbolRef7getNameEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='186' c='_ZNK4llvm6object9SymbolRef10getAddressEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='190' c='_ZNK4llvm6object9SymbolRef8getValueEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='195' c='_ZNK4llvm6object9SymbolRef7getTypeEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='199' c='_ZNK4llvm6object9SymbolRef10getSectionEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='245' c='_ZNK4llvm6object10ObjectFile13getSymbolNameENS0_11DataRefImplE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='248' c='_ZNK4llvm6object10ObjectFile16getSymbolAddressENS0_11DataRefImplE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='252' c='_ZNK4llvm6object10ObjectFile13getSymbolTypeENS0_11DataRefImplE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='253' c='_ZNK4llvm6object10ObjectFile16getSymbolSectionENS0_11DataRefImplE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='260' c='_ZNK4llvm6object10ObjectFile14getSectionNameENS0_11DataRefImplE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='264' c='_ZNK4llvm6object10ObjectFile18getSectionContentsENS0_11DataRefImplE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='280' c='_ZNK4llvm6object10ObjectFile19getRelocatedSectionENS0_11DataRefImplE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='291' c='_ZNK4llvm6object10ObjectFile14getSymbolValueENS0_11DataRefImplE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='298' c='_ZNK4llvm6object10ObjectFile19getCommonSymbolSizeENS0_11DataRefImplE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='332' c='_ZNK4llvm6object10ObjectFile15getStartAddressEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='349' c='_ZN4llvm6object10ObjectFile16createObjectFileENS_9StringRefE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='352' c='_ZN4llvm6object10ObjectFile16createObjectFileENS_15MemoryBufferRefENS_10file_magicEb'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='355' c='_ZN4llvm6object10ObjectFile16createObjectFileENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='364' c='_ZN4llvm6object10ObjectFile20createCOFFObjectFileENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='367' c='_ZN4llvm6object10ObjectFile21createXCOFFObjectFileENS_15MemoryBufferRefEj'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='370' c='_ZN4llvm6object10ObjectFile19createELFObjectFileENS_15MemoryBufferRefEb'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='373' c='_ZN4llvm6object10ObjectFile21createMachOObjectFileENS_15MemoryBufferRefEjj'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='378' c='_ZN4llvm6object10ObjectFile20createWasmObjectFileENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='386' c='_ZNK4llvm6object9SymbolRef7getNameEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='390' c='_ZNK4llvm6object9SymbolRef10getAddressEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='394' c='_ZNK4llvm6object9SymbolRef8getValueEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='406' c='_ZNK4llvm6object9SymbolRef10getSectionEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='410' c='_ZNK4llvm6object9SymbolRef7getTypeEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='443' c='_ZNK4llvm6object10SectionRef7getNameEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='459' c='_ZNK4llvm6object10SectionRef11getContentsEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='460' c='_ZNK4llvm6object10SectionRef11getContentsEv'/>
<use f='halide/llvm-install/include/llvm/Object/ObjectFile.h' l='519' c='_ZNK4llvm6object10SectionRef19getRelocatedSectionEv'/>
<use f='halide/llvm-install/include/llvm/ExecutionEngine/JITSymbol.h' l='187' c='_ZN4llvm14JITSymbolFlags16fromObjectSymbolERKNS_6object9SymbolRefE'/>
<use f='halide/llvm-install/include/llvm/ExecutionEngine/JITSymbol.h' l='267'/>
<use f='halide/llvm-install/include/llvm/ExecutionEngine/JITSymbol.h' l='341' c='_ZN4llvm9JITSymbol10getAddressEv'/>
<use f='halide/llvm-install/include/llvm/ExecutionEngine/JITSymbol.h' l='375'/>
<use f='halide/llvm-install/include/llvm/ExecutionEngine/JITSymbol.h' l='390' c='_ZN4llvm17JITSymbolResolver20getResponsibilitySetERKSt3setINS_9StringRefESt4lessIS2_ESaIS2_EE'/>
<use f='halide/llvm-install/include/llvm/ExecutionEngine/JITSymbol.h' l='407' c='_ZN4llvm23LegacyJITSymbolResolver20getResponsibilitySetERKSt3setINS_9StringRefESt4lessIS2_ESaIS2_EE'/>
<use f='halide/llvm-install/include/llvm/ADT/APFloat.h' l='308' c='_ZN4llvm6detail9IEEEFloat17convertFromStringENS_9StringRefENS_12RoundingModeE'/>
<use f='halide/llvm-install/include/llvm/ADT/APFloat.h' l='536' c='_ZN4llvm6detail9IEEEFloat28convertFromHexadecimalStringENS_9StringRefENS_12RoundingModeE'/>
<use f='halide/llvm-install/include/llvm/ADT/APFloat.h' l='537' c='_ZN4llvm6detail9IEEEFloat24convertFromDecimalStringENS_9StringRefENS_12RoundingModeE'/>
<use f='halide/llvm-install/include/llvm/ADT/APFloat.h' l='664' c='_ZN4llvm6detail13DoubleAPFloat17convertFromStringENS_9StringRefENS_12RoundingModeE'/>
<use f='halide/llvm-install/include/llvm/ADT/APFloat.h' l='1132' c='_ZN4llvm7APFloat17convertFromStringENS_9StringRefENS_12RoundingModeE'/>
<use f='halide/llvm-install/include/llvm/IR/DataLayout.h' l='237' c='_ZN4llvm10DataLayout5parseENS_9StringRefE'/>
<use f='halide/llvm-install/include/llvm/Target/TargetMachine.h' l='451' c='_ZN4llvm17LLVMTargetMachine16createMCStreamerERNS_17raw_pwrite_streamEPS1_NS_15CodeGenFileTypeERNS_9MCContextE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='44' c='_ZN4llvm30expectedToErrorOrAndEmitErrorsERNS_11LLVMContextENS_8ExpectedIT_EE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='80' c='_ZN4llvm22getBitcodeFileContentsENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='83' c='_ZN4llvm13BitcodeModule13getModuleImplERNS_11LLVMContextEbbbNS_12function_refIFNS_8OptionalINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEEENS_9StringRefEEEE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='101' c='_ZN4llvm13BitcodeModule13getLazyModuleERNS_11LLVMContextEbb'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='106' c='_ZN4llvm13BitcodeModule11parseModuleERNS_11LLVMContextENS_12function_refIFNS_8OptionalINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEEENS_9StringRefEEEE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='112' c='_ZN4llvm13BitcodeModule10getLTOInfoEv'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='115' c='_ZN4llvm13BitcodeModule10getSummaryEv'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='133' c='_ZN4llvm22getBitcodeFileContentsENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='136' c='_ZN4llvm20getBitcodeModuleListENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='143' c='_ZN4llvm20getLazyBitcodeModuleENS_15MemoryBufferRefERNS_11LLVMContextEbb'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='152' c='_ZN4llvm26getOwningLazyBitcodeModuleEOSt10unique_ptrINS_12MemoryBufferESt14default_deleteIS1_EERNS_11LLVMContextEbb'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='159' c='_ZN4llvm22getBitcodeTargetTripleB5cxx11ENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='163' c='_ZN4llvm31isBitcodeContainingObjCCategoryENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='168' c='_ZN4llvm24getBitcodeProducerStringB5cxx11ENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='171' c='_ZN4llvm16parseBitcodeFileENS_15MemoryBufferRefERNS_11LLVMContextENS_12function_refIFNS_8OptionalINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEEENS_9StringRefEEEE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='178' c='_ZN4llvm17getBitcodeLTOInfoENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='181' c='_ZN4llvm21getModuleSummaryIndexENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Bitcode/BitcodeReader.h' l='193' c='_ZN4llvm28getModuleSummaryIndexForFileENS_9StringRefEb'/>
<use f='halide/llvm-install/include/llvm/Support/FileSystem.h' l='831' c='_ZN4llvm3sys2fs8TempFile6createERKNS_5TwineEj'/>
<use f='halide/llvm-install/include/llvm/Support/FileSystem.h' l='949' c='_ZN4llvm3sys2fs14openNativeFileERKNS_5TwineENS1_19CreationDispositionENS1_10FileAccessENS1_9OpenFlagsEj'/>
<use f='halide/llvm-install/include/llvm/Support/FileSystem.h' l='981' c='_ZN4llvm3sys2fs14readNativeFileEiNS_15MutableArrayRefIcEE'/>
<use f='halide/llvm-install/include/llvm/Support/FileSystem.h' l='992' c='_ZN4llvm3sys2fs19readNativeFileSliceEiNS_15MutableArrayRefIcEEm'/>
<use f='halide/llvm-install/include/llvm/Support/FileSystem.h' l='1031' c='_ZN4llvm3sys2fs22openNativeFileForWriteERKNS_5TwineENS1_19CreationDispositionENS1_9OpenFlagsEj'/>
<use f='halide/llvm-install/include/llvm/Support/FileSystem.h' l='1073' c='_ZN4llvm3sys2fs26openNativeFileForReadWriteERKNS_5TwineENS1_19CreationDispositionENS1_9OpenFlagsEj'/>
<use f='halide/llvm-install/include/llvm/Support/FileSystem.h' l='1110' c='_ZN4llvm3sys2fs21openNativeFileForReadERKNS_5TwineENS1_9OpenFlagsEPNS_15SmallVectorImplIcEE'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='46' c='_ZNK4llvm6object19ArchiveMemberHeader10getRawNameEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='49' c='_ZNK4llvm6object19ArchiveMemberHeader7getNameEm'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='51' c='_ZNK4llvm6object19ArchiveMemberHeader7getSizeEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='53' c='_ZNK4llvm6object19ArchiveMemberHeader13getAccessModeEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='54' c='_ZNK4llvm6object19ArchiveMemberHeader15getLastModifiedEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='61' c='_ZNK4llvm6object19ArchiveMemberHeader6getUIDEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='62' c='_ZNK4llvm6object19ArchiveMemberHeader6getGIDEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='98' c='_ZNK4llvm6object7Archive5Child12isThinMemberEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='110' c='_ZNK4llvm6object7Archive5Child7getNextEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='112' c='_ZNK4llvm6object7Archive5Child7getNameEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='113' c='_ZNK4llvm6object7Archive5Child11getFullNameB5cxx11Ev'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='114' c='_ZNK4llvm6object7Archive5Child10getRawNameEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='116' c='_ZNK4llvm6object7Archive5Child15getLastModifiedEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='124' c='_ZNK4llvm6object7Archive5Child6getUIDEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='125' c='_ZNK4llvm6object7Archive5Child6getGIDEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='127' c='_ZNK4llvm6object7Archive5Child13getAccessModeEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='132' c='_ZNK4llvm6object7Archive5Child7getSizeEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='134' c='_ZNK4llvm6object7Archive5Child10getRawSizeEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='136' c='_ZNK4llvm6object7Archive5Child9getBufferEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='140' c='_ZNK4llvm6object7Archive5Child18getMemoryBufferRefEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='142' c='_ZNK4llvm6object7Archive5Child11getAsBinaryEPNS_11LLVMContextE'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='194' c='_ZNK4llvm6object7Archive6Symbol9getMemberEv'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='222' c='_ZN4llvm6object7Archive6createENS_15MemoryBufferRefE'/>
<use f='halide/llvm-install/include/llvm/Object/Archive.h' l='258' c='_ZNK4llvm6object7Archive7findSymENS_9StringRefE'/>
<use f='halide/llvm-install/include/llvm/Object/ArchiveWriter.h' l='29' c='_ZN4llvm16NewArchiveMember12getOldMemberERKNS_6object7Archive5ChildEb'/>
<use f='halide/llvm-install/include/llvm/Object/ArchiveWriter.h' l='32' c='_ZN4llvm16NewArchiveMember7getFileENS_9StringRefEb'/>
<use f='halide/llvm-install/include/llvm/Object/ArchiveWriter.h' l='36' c='_ZN4llvm26computeArchiveRelativePathB5cxx11ENS_9StringRefES0_'/>
<use f='halide/llvm-install/include/llvm/Object/ArchiveWriter.h' l='45' c='_ZN4llvm20writeArchiveToBufferENS_8ArrayRefINS_16NewArchiveMemberEEEbNS_6object7Archive4KindEbb'/>
<doc f='halide/llvm-install/include/llvm/Support/Error.h' l='433'>/// Tagged union holding either a T or a Error.
///
/// This class parallels ErrorOr, but replaces error_code with Error. Since
/// Error cannot be copied, this class replaces getError() with
/// takeError(). It also adds an bool errorIsA&lt;ErrT&gt;() method for testing the
/// error class type.</doc>
<smbr r='llvm::Expected::isRef' t='const bool'/>
<fun r='_ZN4llvm8ExpectedC1ENS_5ErrorE'/>
<fun r='_ZN4llvm8ExpectedC1ENS_12ErrorSuccessE'/>
<fun r='_ZN4llvm8ExpectedC1EOTL0__PNSt9enable_ifIXsr3std14is_convertibleIS1_T_EE5valueEvE4typeE'/>
<fun r='_ZN4llvm8ExpectedC1EONS_8ExpectedIT_EE'/>
<fun r='_ZN4llvm8ExpectedC1EONS_8ExpectedITL0__EEPNSt9enable_ifIXsr3std14is_convertibleIS2_T_EE5valueEvE4typeE'/>
<fun r='_ZN4llvm8ExpectedC1EONS_8ExpectedITL0__EEPNSt9enable_ifIXntsr3std14is_convertibleIS2_T_EE5valueEvE4typeE'/>
<fun r='_ZN4llvm8ExpectedaSEONS_8ExpectedIT_EE'/>
<fun r='_ZN4llvm8ExpectedD1Ev'/>
<fun r='_ZN4llvm8ExpectedcvbEv'/>
<fun r='_ZN4llvm8Expected3getEv'/>
<fun r='_ZNK4llvm8Expected3getEv'/>
<fun r='_ZNK4llvm8Expected8errorIsAEv'/>
<fun r='_ZN4llvm8Expected9takeErrorEv'/>
<fun r='_ZN4llvm8ExpectedptEv'/>
<fun r='_ZNK4llvm8ExpectedptEv'/>
<fun r='_ZN4llvm8ExpecteddeEv'/>
<fun r='_ZNK4llvm8ExpecteddeEv'/>
<fun r='_ZN4llvm8Expected21compareThisIfSameTypeERKTL0__S3_'/>
<fun r='_ZN4llvm8Expected21compareThisIfSameTypeERKTL0__RKTL0_0_'/>
<fun r='_ZN4llvm8Expected13moveConstructEONS_8ExpectedITL0__EE'/>
<fun r='_ZN4llvm8Expected10moveAssignEONS_8ExpectedITL0__EE'/>
<fun r='_ZN4llvm8Expected9toPointerEPNSt16remove_referenceIT_E4typeE'/>
<fun r='_ZNK4llvm8Expected9toPointerEPKNSt16remove_referenceIT_E4typeE'/>
<fun r='_ZN4llvm8Expected9toPointerEPSt17reference_wrapperINSt16remove_referenceIT_E4typeEE'/>
<fun r='_ZNK4llvm8Expected9toPointerEPKSt17reference_wrapperINSt16remove_referenceIT_E4typeEE'/>
<fun r='_ZN4llvm8Expected10getStorageEv'/>
<fun r='_ZNK4llvm8Expected10getStorageEv'/>
<fun r='_ZN4llvm8Expected15getErrorStorageEv'/>
<fun r='_ZNK4llvm8Expected15getErrorStorageEv'/>
<fun r='_ZN4llvm8Expected12setUncheckedEv'/>
<fun r='_ZN4llvm8Expected15assertIsCheckedEv'/>
<mbr r='llvm::Expected::HasError' t='bool'/>
<use f='halide/src/Introspection.cpp' l='929' c='_ZN6Halide8Internal13Introspection12_GLOBAL__N_113DebugSections26load_and_parse_object_fileERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE'/>
<use f='halide/src/LLVM_Output.cpp' l='151' c='_ZN6Halide8Internal7Archive12_GLOBAL__N_118write_symbol_tableERSoRKSt6vectorIN4llvm16NewArchiveMemberESaIS6_EEbPSt3mapImS4_INS2_9PatchInfoESaISC_EESt45819532'/>
<use f='halide/src/LLVM_Output.cpp' l='554' c='_ZN6Halide21create_static_libraryERKSt6vectorINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEESaIS6_EERKNS_6TargetERKS6_b'/>
